Slack-Apps/week4/homework/responses/4.1-express.md

```diff
@@ -0,0 +1,19 @@
+# ⚡️Express
+
+This week will involve using a popular JavaScript framework called Express. Frameworks give you a fundamental structure that you can build on without starting completely from scratch. For example, Bolt is also a framework. 
+
+
+
+## 📚Reading Up
+
+Express is a pretty large framework, so you will need to do a little reading to understand its basic functions. Look at the Express-related sections in [this article](https://codeforgeek.com/express-nodejs-tutorial/) for some general information.
+
+After you are done, read [this article](https://codeforgeek.com/handle-get-post-request-express-4/) on POST and GET requests. Comment one idea for a Slack app using Express and a couple of things that stood out to you or you had questions on while researching the framework. 
+
+
+
+## 🔨Challenge
+
+Create a post request that prints a hello message to your local host server. Create another post request that takes parameters for a book and author, then prints out a sentence using the information.
```

Comment:
Are you going to ask students to commit the code for this? How are mentors going to know that students completed this?